terraform/environments/core-logging/monitoring.tf

```diff
@@ -2,3 +2,44 @@ module "core_monitoring" {
   source                     = "../../modules/core-monitoring"
   pagerduty_integration_keys = local.pagerduty_integration_keys
 }
+
+provider "aws" {
+  alias  = "eu-west-2"
+  region = "eu-west-2"
+}
+
+resource "aws_cloudwatch_log_group" "aws_route53_logs_com" {
```

Comment:
Does a provider need to be hardcoded? Isn't the default provider sufficient?

Reply:
Copy and pasted the resource from terraform but forgot to remove the provider